components/projectQueryForm.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import firebase from 'firebase'
+import moment from 'moment'
+import 'common/initializeFirebase.js'
+import Div from 'components/core/div.js'
+import Button from 'components/core/button.js'
+
+export default class ProjectForm extends Component {
+  constructor() {
+    super()
+    this.state = { name: '', email: '', message: '' }
+  }
+
+  updateName(value) {
+    this.setState({ name: value })
+  }
+
+  updateEmail(value) {
+    this.setState({ email: value })
+  }
+
+  updateMessage(value) {
+    this.setState({ message: value })
+  }
+
+  sendMessage(message) {
+    firebase.database().ref('/projectQueries').update(message)
+    this.setState({ name: '', email: '', message: '' })
+  }
+
+  render() {
+    console.log(this.state)
+    return (
+      <Div className="container">
+        <div className="row">
```

Comment:
if you look ahead in the trellos we're trying to eliminate bootstrap. So can we redo this html without using any bootstrap classes just use flexbox instead.